### DISCUSSION TITLE
U13にLEDを実装すると見やすくなります！

### DISCUSSION THREAD

**Discussion by nakkyi** [algyan/XIAOGYAN#73] [Ideas]
ナッキィー（プログラム初心者）と申します。
エンコーダーの内側にあるLEDが見にくいので、U13にＬＥＤを追加をしました。
そして、R12に１KΩの抵抗を付けました。
![DSC_0507](https://github.com/algyan/XIAOGYAN/assets/47250321/7645c097-be23-4533-8d19-402b804b6de7)
これで、ＬＥＤがよく見える様になりました。
https://youtu.be/4Qj8zkUNMhY

**Comment by matsujirushi** [comment]
@nakkyi 
投稿、ありがとうございます！
おっしゃる通り、ロータリーエンコーダー内蔵LEDは見にくいですね。。。

XIAOGYANは完成品ではなく、[改造したり情報交換して楽しむもの](https://github.com/algyan/XIAOGYAN/blob/main/manuals/guidebook/README.md#%E6%A5%BD%E3%81%97%E3%81%BF%E6%96%B9)なので、どしどし手を加えてみてください！

**Comment by kurokouji** [comment]
ロータリーエンコーダー内蔵LEDを明るく光らせるために510Ωの抵抗を並列接続しました。
光量は十分になりましたが、3日ほど連続運用した結果、XiaoESP32C3が不安定になり、最終的に焼損してしまいました。
ロータリーエンコーダー内蔵LEDに十分な電流を流すためには回路にスイッチング用のトランジスを追加したほうが良さそうです。
